dynamic/coinchange.go

```diff
@@ -0,0 +1,21 @@
+package dynamic
+
+// return combination of coin
+func CoinChange(coins []int32, amount int32) int32 {
+	combination := make([]int32, amount)
+	combination[0] = 1
+
+	//Time Complexity O(mn)
```

Comment:
These comments are unnecessary.

Reply:
can I change the code directly here, without having to open a pull request again?.

Reply:
Yeah of course.